### PR TITLE
Checkout style adaptions

### PR DESCRIPTION
**Pull request by danimoh**
This PR is based on `soeren/new-checkout` and also targeting that branch.

If I find the time, I'll add more changes to this PR.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
         "@nimiq/keyguard-client": "^0.1.2",
         "@nimiq/network-client": "^0.1.3",
         "@nimiq/rpc": "^0.0.11",
-        "@nimiq/style": "^0.4.0",
+        "@nimiq/style": "https://github.com/nimiq/nimiq-style.git#julian/hover-states",
```
Comment (sisou) [inline package.json]: `v.0.4.1` of this package has been released now.

File: src/App.vue [hunk 2]
```diff
@@ -93,7 +93,7 @@ export default class App extends Vue {
     .logo .nimiq-logo {
         height: 4rem;
         width: 4rem;
-        margin-right: 2rem;
+        margin-right: 1rem;
```
Comment (sisou) [inline src/App.vue]: The styleguide says, the gap between the signet and the NIMIQ wordmark should be 0.382 times the signet width. So if the signet is 4rem wide, the margin should be `0.382 * 4rem = 1.528rem`.

File: src/views/Checkout.vue [hunk 3]
```diff
@@ -330,7 +330,11 @@ export default class Checkout extends Vue {
         transition: opacity 300ms, z-index 300ms;
     }
 
-    .account-info.active {
+    .merchant-info-shown > :not(.account-info) {
+        filter: blur(.75rem);
```
Comment (sisou) [inline src/views/Checkout.vue]: @nibhar I think you discussed with Julian the exact blur amounts for the various elements. Can you help out here?

File: src/views/Checkout.vue [hunk 4]
```diff
@@ -367,5 +371,7 @@ export default class Checkout extends Vue {
 <style>
     #logo-checkout {
         margin-left: 0.75rem;
+        text-transform: none;
+        letter-spacing: initial;
```
Comment (sisou) [inline src/views/Checkout.vue]: These rules probably don't hurt, but why are they necessary?

File: src/App.vue (hunk 2 above)
Comment (danimoh) [reply src/App.vue]: Yes, you're right. Actually the logo text fixes were a quick fix to roughly match an SVG logo Julian sent me.
The logo label does now perfectly match that svg.

File: src/views/Checkout.vue (hunk 4 above)
Comment (danimoh) [reply src/views/Checkout.vue]: `text-transform` and `letter-spacing` are set on the parent with class `logo` to non-initial values.
I moved this to `App.vue` as a general style rule now.

**Pull request merged**